src/components/EventForm/Form.js

```diff
@@ -0,0 +1,194 @@
+                📲 Please bring your Mac or PC to participate in the workshop.
+                Event Details:
+                😁 Who: YOU, any students of any faculty who are interested in learning Python, and professional instructors from Lighthouse Labs
+                🏢 Where: Henry Angus Building, Room 295
+                📅 When: Monday, November 6th from 5:30 - 8:30pm
+                💸 Cost: This event is FREE for BizTech Members. Non-members can purchase membership upon sign-in at the event.
+            </Typography>
+            <h4 style={{color: 'red'}}>
+                *Required
+            </h4>
+            <Grid container spacing={3}>
+          
+                <Grid item xs={12} sm={6}>
+                    <TextField
+                        label="First Name*"
+                        autoComplete="given-name"
+                        helperText={touched.firstname ? errors.firstname : ""}
+                        error={touched.firstname && Boolean(errors.firstname)}
+                        id="firstname"
+                        value={firstname}
+                        onChange={change.bind(null, "firstname")}
+                        onKeyPress={e => handleKey(e)} 
+                        InputLabelProps={{style: {fontSize: textFieldLabelFontSize}}}
+                        fullWidth
+                    />       
+                </Grid>
+                <Grid item xs={12} sm={6}>
+                    <TextField
+                        label="Last Name*"
+                        autoComplete="family-name"
+                        helperText={touched.lastname ? errors.lastname : ""}
+                        error={touched.lastname && Boolean(errors.lastname)}
+                        id="lastname"
+                        value={lastname}
+                        onChange={change.bind(null, "lastname")}
+                        onKeyPress={e => handleKey(e)} 
+                        InputLabelProps={{style: {fontSize: textFieldLabelFontSize}}}
+                        fullWidth
+                    />       
+                </Grid>
+                <Grid item xs={12}>
+                    <TextField
+                        label="Email Address*"
+                        autoComplete="email"
+                        helperText={touched.email ? errors.email : ""}
+                        error={touched.email && Boolean(errors.email)}
+                        id="email"
+                        value={email}
+                        onChange={change.bind(null, "email")}
+                        onKeyPress={e => handleKey(e)} 
+                        InputLabelProps={{style: {fontSize: textFieldLabelFontSize}}}
+                        fullWidth
+                    />
+                </Grid>
+                {/* TODO: make required for my custom radio group. TODO: make other field in radio buttons */}
+                <Grid item xs={12} sm={6}>
```

Comment:
Actually can we make the radio button sections below each other instead of next to each other?

Reply:
It wouldn't look as compact on a PC, but on a phone that may work better. I can add this.

Reply:
If you view dev.aluu.me (and go to the Dev) tab on a phone, it actually formats so it's below each other. Do you still want the change to be permanently vertical only?

Reply:
Yeah